parent/pom.xml

```diff
@@ -148,6 +149,31 @@
                     <artifactId>maven-gpg-plugin</artifactId>
                     <version>${version.plugin.gpg}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-site-plugin</artifactId>
+                    <version>${version.plugin.site}</version>
+                    <configuration>
+                        <skip>true</skip>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>site-jar</id>
+                            <goals>
+                                <goal>jar</goal>
+                            </goals>
+                            <phase>site</phase>
+                            <configuration>
+                                <archiveExcludes>
+                                    <exclude>.asciidoctor/**</exclude>
+                                    <exclude>css/maven-*.css</exclude>
```

Comment:
The css excludes are not needed anymore, I've added code in the site renderer to exclude these.

Reply:
Fixed